src/index.html

```diff
@@ -16,17 +16,38 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search__form"
+      data-qa="big"
+    >
+      <label for="big-search-input">
+        <input
+          id="big-search-input"
+          class="search__input"
+          type="text"
+          placeholder="Try “Los Angeles“"
+          data-qa="keypress"
+          name="search__form"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search__form"
+      data-qa="small"
+    >
+      <label for="small-search-input">
+        <input
+          id="small-search-input"
+          class="search__input"
+          type="text"
+          placeholder="Try “Los Angeles“"
+          data-qa="hover"
+          name="search__form"
+        />
```

Comment:
HTML Formatting: The attributes of the `input` tag should be correctly formatted. Each attribute should start on a new line with 2-space indentation related to the tag.